xpdacq/xpdacq.py

```diff
@@ -232,6 +232,9 @@ def _inject_filter_positions(msg):
     if msg.command == 'open_run':
         filter_bank = xpd_configuration['filter_bank']
         filters = ['flt1', 'flt2', 'flt3', 'flt4']
+        print("INFO: Current filter status")       
+        for el in filters:
+            print("INFO: {} : {}".format(el, getattr(filter_bank, el).value))
```

Comment:
this is good.  Are we also saving this info by default as run metadata?  It may be in another PR, but I think that is behavior that we want.  Not just a print statement.

Reply:
Yes, we inject the filter position into ``run_start`` metadata as well. Print statement is for experimenter.